### PR TITLE
theme & CSS docs

### PR DESCRIPTION
**Pull request by sarahghp**
Feel free to move and change as needed. With luck this is a good base.

### CODE REVIEW

**Review by SaraVieira — APPROVED** [review]
This looks great!

File: docs/src/options/theme.md [hunk 1]
```diff
@@ -1,18 +1,92 @@
 # Theme
 
-By default Oranda comes with four default themes.
+Oranda comes with two default themes:
 
 ## ADD IMAGES UNDER EACH
```
Comment (ashleygwilliams) [inline docs/src/options/theme.md]: should we hold off merging until we get images here?

Comment (ashleygwilliams) [reply docs/src/options/theme.md]: we could also publish 2 example pages to show off the themes

**Review by ashleygwilliams — APPROVED** [review]

**Review by ashleygwilliams — COMMENTED** [review]
this is great! thank you!

**Pull request merged**